namespace.go

```diff
@@ -60,6 +63,7 @@ type (
 	Namespace struct {
 		Name          string
 		Suffix        string
+		ResourceURI   string
```

Comment:
Is this field actually needed, can't we just get the value from `Environment`?

Reply:
yes indeed.
since we are able to Set custom environment, we still have a way to provide custom values. 
that's the same with Suffix, but would be a breaking change.